clients/params.go

```diff
@@ -18,3 +20,24 @@ type OrgBucketParams struct {
 	OrgParams
 	BucketParams
 }
+
+type AuthLookupParams struct {
+	ID       influxid.ID
+	Username string
+	Required bool // Required when set to true determines whether validate expects either ID or Username to be set
```

Comment:
Instead of putting this in the struct, could we remove it and not call `Validate()` in contexts where the args aren't required?